assembl/static2/js/app/pages/administration.jsx

```diff
@@ -311,10 +312,19 @@ const isNotInLandingPageAdmin = isNotInAdminSection('landingPage');
 export default compose(
   connect(mapStateToProps, mapDispatchToProps),
   graphql(VoteSessionQuery, {
-    skip: ({ timeline }) => typeof getPhaseId(timeline, 'voteSession') !== 'string',
-    options: ({ timeline, i18n }) => {
-      const id = timeline ? getPhaseId(timeline, 'voteSession') : null;
-      const discussionPhaseId = id ? atob(id).split(':')[1] : null;
+    skip: ({ location, timeline }) => {
+      if (!location.query.phaseId) {
+        return true;
+      }
+      const phase = getPhaseById(timeline, location.query.phaseId);
+      if (!phase) {
+        return true;
+      }
+      return phase.identifier !== 'voteSession';
```

Comment:
These phase names have constants now. Use it here.

Reply:
Sure. @amensouissi can you use the constant here please?